content/english/_index.html

```diff
@@ -14,8 +14,88 @@
 
 {{< button label="Read the full story" link="/history/" outline=true >}}
-{{< /promo >}}
\ No newline at end of file
+{{< /image-promo >}}
+
+
+
+{{< icon-promo title="Stay up to date." icon="mastodon" right=true >}}
+We post news and events on a regular base on our social media. Follow us on
+
+{{< button external=true label="Mastodon" link="http://social.hedgedoc.org/" >}}
+{{< /icon-promo >}}
+
+
+
+
+{{< icon-promo background="secondary" title="Let's get it started!" icon="rocket" >}}
+Installing HedgeDoc is easy! We provide a precompiled build and a docker image. If you need help, then feel free to read our documentation.
+
+{{< button external=true label="Getting started" link="https://github.com/hedgedoc/hedgedoc/#installation--upgrading" >}}
+{{< /icon-promo >}}
+
+
+
+
+{{< icon-promo title="Read the docs." icon="wrench" right=true >}}
+How do I configure OAuth? Can I restrict edits to logged in users? Read our docs and make your HedgeDoc instance fitting perfectly for your use case.
+
+{{< button external=true label="Read the documentation" link="https://docs.hedgedoc.org" >}}
+{{< /icon-promo >}}
+
+
+
+{{< icon-promo background="primary" title="Open Source is king." icon="code-fork" >}}
+The source code of hedgedoc and every tool is available on GitHub licensed under AGPL 3.0. Everything is open source and free as in free speech.
+Read it, understand it, improve it!
+
+{{< button external=true label="Visit our GitHub repository" link="https://github.com/hedgedoc/hedgedoc" >}}
+{{< /icon-promo >}}
+
+
+
+
+
+{{< icon-promo title="Contact us, Discuss with us, Ask us." icon="commenting" right=true >}}
+We are always open for improvements, wishes and discussions. You can join our Matrix Room or open a topic in our Discourse.
+Maybe your question has already been covered by our FAQ.
+
+{{< button external=true label="Join our Matrix room" link="https://chat.hedgedoc.org" >}}
+{{< button external=true label="Visit our Discourse" link="https://community.hedgedoc.org" >}}
+{{< button label="Visit the FAQ" link="/faq/" >}}
+{{< /icon-promo >}}
+
+
+
+
+
+{{< icon-promo background="secondary" title="Help with translations." icon="flag" >}}
+¿Hablas español? Kan du norsk? 你会说中文吗? tlhIngan Hol Dajatlh'a'?
+You don't have to be a programmer to improve HedgeDoc. If you speak another language than english, then you can help us with translation.
+
+{{< button external=true label="Visit POEditor" link="http://translate.hedgedoc.org/" >}}
+{{< /icon-promo >}}
+
+
+
+
+
+{{< icon-promo title="Meet our Team." icon="users" right=true >}}
+Behind the scenes are working many many volunteers on this project. Come and meet our core development team!
+
+{{< button label="Visit the team page" link="/team/" >}}
+{{< /icon-promo >}}
+
+
+
+
+{{< icon-promo background="primary" title="Try the demo." icon="child" >}}
+Why should you use something you don't know, right? Head over to our demo page and test it out. Try it! It's free!
```

Comment:
Instead of the “Try it, it's free!” it could be good to have a link to the FAQ entry about the demo stability here.

Reply:
Added a button